### PR TITLE
Removing dependency. Storage does not depend on tests now.

### PR DESCRIPTION
**Pull request by bykoianko**
https://trello.com/c/QcIMhU0f/2232-storage-testing

### CODE REVIEW

File: storage/storage_tests/storage_tests.cpp [hunk 1]
```diff
@@ -845,7 +845,7 @@ UNIT_TEST(StorageTest_GetRootId)
                            "id": "Countries",
                            "v": 151227,
                            "g": []
-                         })"));
+                         })"), new TestMapFilesDownloader());
```
Comment (ygorshenin) [inline storage/storage_tests/storage_tests.cpp]: Memory leak here, right?

Comment (ygorshenin) [inline storage/storage_tests/storage_tests.cpp]: OK, no leak, but in any case, please, pass test downloader as unique_ptr, not as raw pointer.

Comment (bykoianko) [inline storage/storage_tests/storage_tests.cpp]: Sure. I'm working on a new PR.

Comment (bykoianko) [inline storage/storage_tests/storage_tests.cpp]: https://github.com/mapsme/omim/pull/1246

**Pull request merged**